website/templates/project/project.mako

```diff
@@ -311,6 +313,7 @@
 </%def>
 
 <%def name="stylesheets()">
+    <link rel="stylesheet" href="/static/css/pages/project-page.css">
```

Comment:
These should probably go after `${parent.stylesheets()}` in case `project-page.css` needs to override any rules from a parent template.